library.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "SevenSegmentTM1637",
+  "version": "1.0.0",
+  "keywords": "TM1637, TM1636, Arduino, LED, AVR, Atmega, esp8266, IC",
+  "description": "Library for using a 4 digit seven segment display with TM1636 or TM1637 driver IC",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/bremme/arduino-tm1637.git"
+  },
+  "authors":
+  [
+    {
+        "name": "Bram Harmsen",
+        "email": "bramharmsen@gmail.com"
+    },
+    {
+        "name": "Andrew Smith",
+        "email": "me@andrew-smith.com",
+        "maintainer": true    	
+    }
+  ],
+  "frameworks": "arduino",
+  "platforms": "avr"
```

Comment:
Think this should be `atmelavr`, but you mentioned esp8266 in the tags, so should probably be at least `atmelavr, espressif8266`... or you can just do the `*` wildcard and see what platforms it doesn't work on :-P :laughing:

Reply:
As you suggested, I used * ;)